extensions/amp-analytics/amp-analytics.md

```diff
@@ -194,6 +194,14 @@ Type attribute value: `webtrekk`
 
 Adds support for Webtrekk. Configuration details can be found at [supportcenter.webtrekk.com](https://supportcenter.webtrekk.com/en/public/amp-analytics.html).
 
+### Linkpulse
+
+Type attribute value: `linkpulse`
+
+Adds support for Linkpulse. Configuration details can be found at [docs.linkpulse.com)(http://docs.linkpulse.com)
```

Comment:
Please fix the link. closing bracket should be a square bracket for the link text.